lxd-benchmark/main.go

```diff
@@ -88,8 +88,8 @@ func main() {
 	app.Example = `  # Spawn 20 Ubuntu containers in batches of 4
   lxd-benchmark launch --count 20 --parallel 4
 
-  # Create 50 Alpine containers in batches of 10
-  lxd-benchmark init --count 50 --parallel 10 images:alpine/edge
+  # Create 50 Ubuntu Minimal 22.04 containers in batches of 10
+  lxd-benchmark init --count 50 --parallel 10 ubuntu-minimal:22.04
```

Comment:
does ubuntu 22.04 work on LXD 4.0 out of interest?

Reply:
Yes it does.